Python/Tests/TestAdapterTests/TestResultParserTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.PythonTools.TestAdapter.Pytest;
+
+namespace TestAdapterTests {
+    /// <summary>
+    /// Summary description for TestResultParserTests
+    /// </summary>
+    [TestClass]
+    public class TestResultParserTests {
+        public TestResultParserTests() {
```

Comment:
You should get rid of all these useless comments and unused code.

Reply:
will do

Reply:
oh you're missing the copyright header too

Reply:
good catch